app/views/courses/show.html.erb

```diff
@@ -96,7 +102,15 @@
 
 <p><%= @course.description %></p>
 
-<p>Prereqs: TODO</p>
+<p>Prereqs: 
+<ul>
+  <% @course.prerequisites.each do |prereq_set| %>
+    <li>
+    <%= prereq_set.map(&:long_string).join(" or ") %>
+    </li>
+  <% end %>
+</ul>
+</p>
```

Comment:
Any reason for all the commented out code down here?

Aside from this, looks good. :+1: